server/apps/abdm/gateway/sessions/services.py

```
@@ -0,0 +1,187 @@
+import uuid
+import requests
+import logging
+from datetime import datetime, timedelta
```

Ruff (0.11.9)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)